### PR TITLE
Drop foreign keys for tables to fix heroku build

### PR DESCRIPTION
**Pull request by sofiiadidovych**
# Description

We changed migration files in order to remove foreign keys, to allow drop tables while seeding process. Also, seeds file for questions was changed. In order to populate question table we need to delete all info from the answers table, and only then we can actually delete info from questions table and insert the new info. This should fix error on Heroku.

Fixes # 132

# How to test?

Heroku build should be successful

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by abedaarabi — APPROVED** [review]
it looks great, just next time push the files that been changes 👍

**Comment by sofiiadidovych** [comment]
> it looks great, just next time push the files that been changes 👍

@abedaarabi Thank you for the review. What do you mean "next time push the files that been changes"? I think I only pushed the 4 files I changed.

File: src/server/seeds/development/01_questions.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 // This is seeding for the questions table.
 exports.seed = function (knex) {
-  return knex('questions')
+  return knex('answers')
```
[the remark this answers is not in the document]
Comment (dpfernandes) [reply src/server/seeds/development/01_questions.js:3]: I can now see that you are replacing it her but you add it underneath :) So 👍

**Review by dpfernandes — APPROVED** [review]
👍

**Pull request merged**